packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md

```diff
@@ -253,6 +253,38 @@ Variables used for function parameters must match the function data type. For ex
 | escapeHtml(String html)                                       | Escapes the characters in the provided string using HTML entities                                                             |
 | escapeHtmlAttr(String html)                                   | Encodes data for use in HTML attributes                                                                           |
 
+## Customize a forgotten password recovery email
+
+Okta can send an email notification to an end user who has forgotten their password. They must follow the provided link to complete the forgot password process.
+
+If you are using an [embedded authentication](/docs/concepts/redirect-vs-embedded/) deployment model for your application's sign-in flow, you can create a customized email magic link for your **Forgot Password** email template.
+
+The option to send a forgotten password recovery email with a custom link is available only to OIDC applications on Identity Engine organizations.
+
+You can customize any default email template. However, when you edit a default template, Okta stops sending the default message in other languages. You are responsible for adding translations of your customized message.
```

Comment:
can we move this paragraph to a top level section if it isn't already apparent? this applies to all email templates, not just this forgot pw template

Reply:
Done :)